roles/ceph-common/tasks/main.yml

```diff
@@ -11,6 +11,23 @@
        cache_valid_time=3600
   with_items: ceph.debian_pkgs
 
+- name: create ceph log directory
+  file: dest=/var/log/ceph
+        state=directory
+        owner=root
+        group=adm
+        mode=2750
+
+- name: Change ceph log directory permissions with acl
```

Comment:
As done with other components, the same logic of creating directory with 2750 and setting the acl on the directory. is followed for Ceph. It worked for all the log files except for "ceph.log" and "ceph.audit.log". These two files, which are located on Ceph Monitor nodes, always ended up with 600 (root:adm) even though they are created before installing the Ceph packages. 
So, ended up adding extra tasks in Monitor role.  The permissions on these files stayed as 644, even after log rotate.